### PR TITLE
[DX-1703] Label-based navigation in the documentation UI

### PR DESCRIPTION
**Pull request by sharadregoti**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<be>

https://deploy-preview-5528--tyk-docs.netlify.app/docs/nightly/apim/

### Description

UX Changes
1. Menu Top Bar
    - Decreased padding from the left side
    - Added a dropdown called `help` & moved all the support menus inside the dropdown
    - Removed the Tab switcher
2. Side Nav
    - Removed the help section from the bottom.
    - The `Tabs` are now rendered as `Labels` inside the side nav
    - Adjusted font size of options inside side nav
    - TODO: Fix SideNav option closing after any option is pressed
3. Main Content
    - Adding padding from left & right to the main content
    - Adjusted font-size of `on this page` menu
    - TODO: Fix font-size & padding

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
enhancement

___

### **Description**
- Refactored the menu rendering logic in `menu_include.html` to improve readability and maintainability, including the introduction of an index counter and conditional rendering for 'Tab' categories.
- Removed the inclusion of the `tabs_menu.html` partial in the header layout, indicating a shift in how tabs are managed or displayed.
- Cleaned up the `menu.yaml` file by removing unused menu items and the "Orphan" tab, streamlining the menu structure.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>header.html</strong><dd><code>Remove tabs menu from header layout.</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/themes/tykio/layouts/partials/header.html

- Commented out the inclusion of `tabs_menu.html`.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5528/files#diff-31e7ac7aef50dea9f482ddf13931cccb517d3c82d65f1b851cdc830c89795b4b">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>menu_include.html</strong><dd><code>Refactor and enhance menu rendering logic.</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/themes/tykio/layouts/partials/menu_include.html

<li>Refactored menu rendering logic.<br> <li> Introduced index counter for menu items.<br> <li> Added conditional rendering for 'Tab' categories.<br> <li> Simplified recursive menu structure.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5528/files#diff-247ccccf890483a3b4c81c115bb695bbcdd17cc52c1eb3b928d20d4780c1f9e5">+82/-62</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>menu.yaml</strong><dd><code>Clean up and remove unused menu items.</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/data/menu.yaml

<li>Removed several menu items under "API Platform Architectures".<br> <li> Deleted "Orphan" tab and its associated pages.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5528/files#diff-a6789c66bfa0660f7fdbe4796002ab4777677bae59a0276689c840100ff0b0b2">+0/-21</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: tyk-docs/data/menu.yaml [hunk 1]
```diff
@@ -1,18 +1,13 @@
 # yamllint disable rule:line-length rule:truthy
 ---
 menu:
-  - title: "Home"
+  - title: "Overview"
```
Comment (letzya) [inline tyk-docs/data/menu.yaml]: @sharadregoti why are we changing it from home to overview if it's the homepage?

Comment (sharadregoti) [reply tyk-docs/data/menu.yaml]: There is no specific reason, we discussed this on a call. 

I can revert it if it doesn't look good.

Comment (letzya) [reply tyk-docs/data/menu.yaml]: please do. (can't remember)

**Review by buger — APPROVED** [review]

**Review by letzya — APPROVED** [review]

**Pull request merged**